### PR TITLE
test: Update ostree.sh to cover bz#1896178

### PR DESCRIPTION
**Pull request by henrywang**
Update details:

1. For RHEL 8.5, user and ssh key are added in blueprint.
2. For Fedora 33, RHEL 8.3 and 8.4, user and ssh key have to be added in ks file due to [bz#1896178](https://bugzilla.redhat.com/show_bug.cgi?id=1896178).

[bz#1896178](https://bugzilla.redhat.com/show_bug.cgi?id=1896178) has been fixed already on RHEL 8.5 only, but still exists on Fedora 33, RHEL 8.3 and 8.4. This PR covers RHEL 8.5, Fedora 33, RHEL 8.4 and RHEL 8.3.

### CODE REVIEW

**Comment by henrywang** [comment]
All ostree tests are green. Failed cases are not related with code change in this PR.

**Review by achilleas-k — DISMISSED** [review]
Looks good.  Thanks!

File: test/cases/ostree.sh [hunk 1]
```diff
@@ -403,6 +426,19 @@ name = "wget"
...
+    tee -a "$BLUEPRINT_FILE" > /dev/null << EOF
+[[customizations.user]]
+name = "${SSH_USER}"
+description = "Administrator account"
+password = "\$6\$GRmb7S0p8vsYmXzH\$o0E020S.9JQGaHkszoog4ha4AQVs3sk8q0DvLjSMxoxHBKnB2FBXGQ/OkwZQfW/76ktHd0NX5nls2LPxPuUdl."
+key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC61wMCjOSHwbVb4VfVyl5sn497qW4PsdQ7Ty7aD6wDNZ/QjjULkDV/yW5WjDlDQ7UqFH0Sr7vywjqDizUAqK7zM5FsUKsUXWHWwg/ehKg8j9xKcMv11AkFoUoujtfAujnKODkk58XSA9whPr7qcw3vPrmog680pnMSzf9LC7J6kXfs6lkoKfBh9VnlxusCrw2yg0qI1fHAZBLPx7mW6+me71QZsS6sVz8v8KXyrXsKTdnF50FjzHcK9HXDBtSJS5wA3fkcRYymJe0o6WMWNdgSRVpoSiWaHHmFgdMUJaYoCfhXzyl7LtNb3Q+Sveg+tJK7JaRXBLMUllOlJ6ll5Hod root@localhost"
+home = "/home/${SSH_USER}/"
+groups = ["wheel"]
+EOF
+fi
```
Comment (atodorov) [inline test/cases/ostree.sh:440]: This block seems to be repeating above. Can we just have only 1 of them ?

**Review by atodorov — APPROVED** [review]
I see one repetition but otherwise looks good to me. Would be better if we can avoid the repetition.

**Comment by henrywang** [comment]
Rebased @achilleas-k's PR that removes the broken tests on 8.5. Everything works now.

**Review by achilleas-k — APPROVED** [review]
Still great!

**Pull request merged**